### ISSUE TITLE
mv: missing destination file operand after ‘/tmp/vagrant_network’

### ISSUE THREAD

**Issue by root-io** [terrywang/vagrantboxes#19]
Got this issue with vagrant 1.7.x

```
$ vagrant up
Bringing machine 'default' up with 'virtualbox' provider...
==> default: Box 'arch64' could not be found. Attempting to find and install...
    default: Box Provider: virtualbox
    default: Box Version: >= 0
==> default: Adding box 'arch64' (v0) for provider: virtualbox
    default: Downloading: http://cloud.terry.im/vagrant/archlinux-x86_64.box
==> default: Box download is resuming from prior download progress
==> default: Successfully added box 'arch64' (v0) for 'virtualbox'!
==> default: Importing base box 'arch64'...
==> default: Matching MAC address for NAT networking...
==> default: Setting the name of the VM: myproject_default_1419353941248_40826
==> default: Clearing any previously set network interfaces...
==> default: Preparing network interfaces based on configuration...
    default: Adapter 1: nat
    default: Adapter 2: hostonly
==> default: Forwarding ports...
    default: 22 => 2222 (adapter 1)
==> default: Running 'pre-boot' VM customizations...
==> default: Booting VM...
==> default: Waiting for machine to boot. This may take a few minutes...
    default: SSH address: 127.0.0.1:2222
    default: SSH username: vagrant
    default: SSH auth method: private key
    default: Warning: Connection timeout. Retrying...
    default: 
    default: Vagrant insecure key detected. Vagrant will automatically replace
    default: this with a newly generated keypair for better security.
    default: 
    default: Inserting generated public key within guest...
    default: Removing insecure key from the guest if its present...
    default: Key inserted! Disconnecting and reconnecting using new SSH key...
==> default: Machine booted and ready!
==> default: Checking for guest additions in VM...
==> default: Configuring and enabling network interfaces...
The following SSH command responded with a non-zero exit status.
Vagrant assumes that this means the command failed!

mv /tmp/vagrant_network
                                     /etc/netctl/

Stdout from the command:



Stderr from the command:

mv: missing destination file operand after ‘/tmp/vagrant_network’
Try 'mv --help' for more information.
bash: line 3: /etc/netctl/: Is a directory
```

**Comment by root-io** [comment]
Issue fixed with vagrant 1.7.2

**Comment by terrywang** [comment]
@root-io Thank you for reporting back ;-D Vagrant 1.7.1 seemed to be very problematic, cannot even properly `vagrant package` a box.
